src/erlcloud_sqs.erl

```diff
@@ -332,11 +347,28 @@ decode_message(Message) ->
        {md5_of_body, "MD5OfBody", text},
        {message_id, "MessageId", text},
        {receipt_handle, "ReceiptHandle", text},
-       {attributes, "Attribute", fun decode_msg_attributes/1}
+       {attributes, "Attribute", fun decode_msg_attributes/1},
+       {message_attributes, "MessageAttribute", fun decode_message_attributes/1}
       ],
       Message
      ).
 
+decode_message_attributes(Attributes) ->
+    [{decode_msg_attribute_name(Name), decode_message_attribute_value(DataType, StringValue)} ||
+        {Name, DataType, StringValue} <- decode_message_attribute(Attributes)].
+
+decode_message_attribute(Attributes) ->
+    [{erlcloud_xml:get_text("Name", Attr),
+      erlcloud_xml:get_text("Value/DataType", Attr),
+      erlcloud_xml:get_text("Value/StringValue", Attr)} || Attr <- Attributes].
+
+decode_message_attribute_value("Number", Value) ->
```

Comment:
There is an issue here with determining if the Number type defined in the response from SQS is a float or an integer. Here, we naively assume Number will always be an integer. Not sure the best way to support coercion to float or integer.